requirements/requirements.txt

```diff
@@ -1,4 +1,5 @@
 # model
+google-cloud-aiplatform
 openai[embeddings]~=0.27
```

Comment:
Is this dependency still used, or just leftover from ChatGPT? I don't see any remaining references